securedrop/source_templates/use-tor-browser.html

```diff
@@ -6,4 +6,5 @@ <h1>{{ gettext('You Should Use Tor Browser') }}</h1>
 <p>{{ gettext('Copy and paste the following address into your browser and follow the instructions to download and install Tor Browser:') }}
   <pre>https://www.torproject.org/projects/torbrowser.html</pre>
 </p>
+<p>{{ gettext('If there is a chance that downloading the Tor Browser raises suspicion and your mail provider is less likely to be monitored, you can send a mail to gettor@torproject.org and a bot will answer with instructions.') }}</p>
```

Comment:
Let's wrap `gettor@torproject.org` in `<pre>` tags so it formats like the URL:

![screen shot 2017-12-12 at 12 32 49 pm](https://user-images.githubusercontent.com/7832803/33907046-ae31cae0-df38-11e7-9559-0d7b0c6574f2.png)